### DISCUSSION TITLE
Favicon of website instead of Anonaddy favicon?

### DISCUSSION THREAD

**Discussion by salmanaligeek** [anonaddy/anonaddy#482] [Q&A]
Hi, 

I was wondering is there any way to show favicon of website instead of favicon of Anonaddy in email clients? If this feature is already implemented then let me know or if it is not then let's do it.

Take a look at screenshot:
![Screenshot_2023-07-17_20-26-32](https://github.com/anonaddy/anonaddy/assets/5575504/74effa3c-033c-4d9d-a50b-34342a765a10)

**Comment by willbrowningme** [comment]
I do not believe so since the messages are forwarded from the alias itself. This would also be seen as a phishing risk if it were possible.
